test/test_qiskit_provider.py

```
@@ -0,0 +1,665 @@
+
+def test_target_selects_platform_by_id(mod):
+    qpu = mod.QPU(platform="ProviderPlatform:b")
+
+    assert qpu.target.num_qubits == 7
+
+
+def test_target_unknown_platform_raises(mod):
+    qpu = mod.QPU(platform="no-such-platform")
+
+    with pytest.raises(LookupError, match="no-such-platform"):
+        qpu.target
```

Ruff (0.15.15)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)